src/SIM/SIM1D.C

```diff
@@ -574,7 +574,13 @@ ASMbase* SIM1D::createDefaultGeometry (const TiXmlElement* geo) const
     g2.append(" 1.0");
   g2.append("\n");
 
-  std::istringstream unitLine(g2);
+  return g2;
+}
+
+
+ASMbase* SIM1D::createDefaultGeometry (const TiXmlElement* geo) const
+{
+  std::istringstream unitLine(createDefaultG2(geo));
   return this->readPatch(unitLine,1,{nf});
```

Comment:
Forslag: Hvis du her bytter om på rekkefølgen av de to metodene, altså definere den nye createDefaultG2 etter createDefaultGeometry, så blir diff'en noe mindre (kun kosmetisk selvsagt). Samme gjelder da for 2D og 3D.